prow/scripts/cluster-integration/kyma-gardener-integration.sh

```diff
@@ -161,10 +161,6 @@ kyma version
 shout "Running Kyma tests"
 date
 echo "- Creating ClusterAddonsConfiguration which provides the testing addons"
```

Comment:
Can you please delete this echo as well?

Reply:
my bad, done :)